app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -233,33 +54,6 @@ function AcuantCaptureCanvas({
     };
   }, []);
 
-  useEffect(() => {
-    if (isReady) {
-      /** @type {AcuantGlobal} */ (window).AcuantCameraUI.start(
-        {
-          onCaptured() {},
-          onCropped,
-        },
-        onImageCaptureFailure,
-        {
-          text: {
-            NONE: t('doc_auth.info.capture_status_none'),
-            SMALL_DOCUMENT: t('doc_auth.info.capture_status_small_document'),
-            GOOD_DOCUMENT: null,
-            CAPTURING: t('doc_auth.info.capture_status_capturing'),
-            TAP_TO_CAPTURE: t('doc_auth.info.capture_status_tap_to_capture'),
-          },
-        },
-      );
-    }
-
-    return () => {
-      if (isReady) {
-        /** @type {AcuantGlobal} */ (window).AcuantCameraUI.end();
-      }
-    };
-  }, [isReady]);
-
   const clickCanvas = () => document.getElementById('acuant-ui-canvas')?.click();
```

Comment:
One thing I realized late is that this component is responsible for rendering the UI that Acuant will assume to be present. With the revisions proposed here, we essentially yank away those elements after the image is captured. At that point, the Acuant instance is still "live" pending the result of the crop. It doesn't appear that this is a problem, but it could be if Acuant were to try to interact with those elements at any point again after the capture is made.

I'm not really concerned enough about it to think it a blocker, but worth noting.

Reply:
This might be even less of an issue than I had initially thought, since on closer inspection of the Acuant source, Acuant will remove these elements / event handlers anyways as soon as the image is captured, _before_ calling `onCaptured`:

https://github.com/Acuant/JavascriptWebSDKV11/blob/9a5387576a33710188501ad6233f986b1b6bb1cb/SimpleHTMLApp/webSdk/dist/AcuantCamera.js#L196-L205